scrapy/exporters.py

```diff
@@ -52,6 +53,17 @@ def start_exporting(self):
     def finish_exporting(self):
         pass
 
+    def _should_export_item(self, item):
+        """Determine whether or not an item should be exported
+        based on the 'items_classes' key in the FEEDS setting
+        """
+        if self.item_classes_to_export is None:
+            return True
+        elif isinstance(item, self.item_classes_to_export):
+            return True
+        else:
+            return False
```

Comment:
```suggestion
        return (
            self.item_classes_to_export is None
            or isinstance(item, self.item_classes_to_export)
        )
```
Shorter and clearer IMHO